plugin/db/mysql/rollback.go

```diff
@@ -35,6 +42,30 @@ func (txn BinlogTransaction) GetRollbackSQL(tables map[string][]string) (string,
 	return strings.Join(sqlList, "\n"), nil
 }
 
+// ParseTableColumns parses the schema to get the table columns map.
+// This is used to generate rollback SQL from the binlog events.
+func ParseTableColumns(schema string) (map[string][]string, error) {
```

Comment:
I'd avoid using Parse() whenever possible from naming perspective. We're doing a Get() instead of Parse().

Reply:
SG. Changed.